### DISCUSSION TITLE
v1.0.6

### DISCUSSION THREAD

**Discussion by dmatik** [dmatik/switcher-boiler-card#1] [Announcements]
Added an option for additional sensor to be displayed in state field.
For example if you a place your own thermostat inside the Boiler to measure actual water temperature.

**Full Changelog**: https://github.com/dmatik/switcher-boiler-card/compare/v1.0.5...v1.0.6

<hr /><em>This discussion was created from the release <a href='https://github.com/dmatik/switcher-boiler-card/releases/tag/v1.0.6'>v1.0.6</a>.</em>

**Comment by tal6203** [comment]
I really appreciate your work and I really liked your card. I would be happy if you would add or enlarge or make some nice animation for the hours of "time remaining" something like a flipdown as an example.
Just a suggestion to improve the clock really small.

**Reply by dmatik** [reply]
Hi,
Thanks for the suggestion. 
The whole point for the card to be same design as HA Tile cards. 

https://www.home-assistant.io/dashboards/tile/
